lms/djangoapps/django_comment_client/utils.py

```diff
@@ -249,11 +249,11 @@ def _sort_map_entries(category_map, sort_alpha):
     for title, entry in category_map["entries"].items():
         if entry["sort_key"] is None and sort_alpha:
             entry["sort_key"] = title
-        things.append((title, entry))
+        things.append((title, entry, 'entry'))
```

Comment:
I know I didn't mention this earlier, but it would be nice to use constants for these.